app/controllers/api_controller.rb

```diff
@@ -12,7 +12,8 @@ def getdeptsbylib
   end
 
   def gethours
```

Comment:
we need to be sure all the 'get' items are out of there.  If it is somsething that needs to be a getter setter style of method then you can use rubies default attr_accessors  and then overwrite the setter.